### ISSUE TITLE
Kerolox Configuration

### ISSUE THREAD

**Issue by Tyaedalis** [ferram4/ProcEngines#6]
@ferram4, if you let me know where your config for kerolox stands so far I will finish it up.

### Parameters:
- **O/F range**: 0.1–60
- **Optimal O/F range**: 2–2.6
- **Frozen Area Ratio**: 1.3
- **Pressure range**: 0.25–100 MPa

### O/F values to calculate:

- [x] 0.1
- [x] 0.5
- [x] 0.8
- [x] 1.0
- [x] 1.2
- [x] 1.5
- [x] 1.9
- [x] 2.0
- [x] 2.1
- [x] 2.2
- [x] 2.3
- [x] 2.4
- [x] 2.5
- [x] 2.6
- [x] 2.7
- [x] 2.8
- [x] 3.0
- [x] 3.5
- [x] 4.0
- [x] 5.0
- [x] 6.0
- [x] 10.0
- [x] 20.0
- [x] 30.0
- [x] 40.0
- [x] 50.0
- [x] 60.0

**Comment by ferram4** [comment]
The config hasn't changed on my end, so you can update it as needed.  The pressures should probably be extended up to 100 MPa though.  O/Fs need to be handled at both extremes, I know that with corrections the lower end can be down at 0.1 and it'll still solve and we need that for proper US gas generator engines.  Make your judgment call on handling the super-ox-rich end, the detailed range is already set except for the higher pressures needed.

**Comment by Tyaedalis** [comment]
Cool.

**Comment by ferram4** [comment]
Bring the higher O/F limit up to around 60.  Russian staged-combustion engines need that kind of range to simulate oxygen-rich preburners.  You shouldn't need too many data points though, the temp should ramp down smoothly on that end.

**Comment by Tyaedalis** [comment]
Got it.

**Comment by Tyaedalis** [comment]
Which, chemical were you using for kerosene, exactly? I'm getting slightly different values for everything I've tried (both RP-1 and T-1). I just want consistent data.

**Comment by ferram4** [comment]
I have been using RP-1.  Because 'murica.  Since RealFuels doesn't distinguish between RP-1, RG-1, T-1, etc. and they're very similar, we'll collapse it all to RP-1.

**Comment by Tyaedalis** [comment]
Okay. I think I'll just start from scratch with this config, for consistency's sake.

Just to show the kind of differences I'm looking at:

Mine:
```
MixtureRatioData
{
	OFratio = 1.2000000
	PressureData
	{	//      Pc,      Tc,        Te,       Pe,     MW,     gamma,  Mach
		key = 0.2500, 1736.6867, 1215.3640, 0.0572, 15.5677, 1.3291, 1.6342
		key = 1.0000, 1736.9763, 1215.3447, 0.2288, 15.5681, 1.3291, 1.6343
		key = 4.0000, 1737.1228, 1215.3398, 0.9154, 15.5683, 1.3291, 1.6343
		key = 6.0000, 1737.1514, 1215.3563, 1.3731, 15.5684, 1.3290, 1.6343
		key = 8.0000, 1737.1702, 1215.4247, 1.8308, 15.5685, 1.3289, 1.6345
		key = 10.0000, 1737.1849, 1215.6374, 2.2888, 15.5686, 1.3282, 1.6351
		key = 15.0000, 1737.2184, 1218.3548, 3.4403, 15.5689, 1.3201, 1.6415
		key = 20.0000, 1737.2615, 1225.6052, 4.6049, 15.5693, 1.3091, 1.6516
		key = 25.0000, 1737.3325, 1235.7574, 5.7972, 15.5700, 1.3059, 1.6534
		key = 30.0000, 1737.4569, 1245.9511, 7.0037, 15.5711, 1.3065, 1.6516
		key = 35.0000, 1737.6703, 1255.4961, 8.2183, 15.5727, 1.3083, 1.6489
		key = 40.0000, 1738.0198, 1264.2563, 9.4362, 15.5754, 1.3106, 1.6463
		key = 50.0000, 1739.3502, 1279.7405, 11.8760, 15.5848, 1.3156, 1.6420
	}
}
```
Yours:
```
MixtureRatioData
{
	OFratio = 1.2
	PressureData
	{  //    Pc,   Tc,        Te,        Pe,     MW,      gamma,  Mach
		key = 0.25, 1736.7313, 1218.5982, 0.0574, 15.5797, 1.3216, 1.6375
		key = 1.00, 1737.6815, 1242.7578, 0.2331, 15.6702, 1.2917, 1.6473
		key = 2.00, 1739.7833, 1270.8181, 0.4739, 15.7825, 1.2820, 1.6441
		key = 4.00, 1746.7642, 1307.4224, 0.9622, 15.9462, 1.2814, 1.6371
		key = 6.00, 1755.7430, 1331.9921, 1.4532, 16.0662, 1.2842, 1.6331
		key = 8.00, 1765.3024, 1350.7030, 1.9452, 16.1623, 1.2873, 1.6306
		key = 10.0, 1774.7707, 1365.9304, 2.4377, 16.2430, 1.2900, 1.6290
		key = 15.0, 1796.6927, 1395.1354, 3.6695, 16.4037, 1.2958, 1.6266
		key = 20.0, 1815.8353, 1417.1328, 4.9032, 16.5286, 1.3002, 1.6251
		key = 25.0, 1832.6195, 1434.9029, 6.1377, 16.6318, 1.3037, 1.6241
		key = 30.0, 1847.5216, 1449.8813, 7.3729, 16.7203, 1.3067, 1.6223
		key = 40.0, 1873.0850, 1474.3015, 9.8425, 16.8678, 1.3113, 1.6223
		key = 50.0, 1894.5400, 1493.9944, 12.3158, 16.9886, 1.3147, 1.6215
	}
}
```

**Comment by ferram4** [comment]
I see, comparing it to my 1.2.  Very strange, not sure where the difference is.  But sure, redo it for consistency's sake.

**Comment by Tyaedalis** [comment]
Whoops, that was using T-1; forgot to change it to RP-1. The differences are even less, but still not the same. I'm gonna do them over.

**Comment by Tyaedalis** [comment]
Ah, I forgot to switch out the thermo database configured for fuel rich calculations. That means UDMH + NTO is inaccurate. How much does that affect things?

**Comment by ferram4** [comment]
At the oxidizer-rich and ideal range it won't affect things much at all.  At the fuel-rich end before it's supposed to exist it'll cause lower temps.  Which is the reason the appropriate behavior there is to simulate the corrected and uncorrected databases and take a weighted average of the results to smooth out the discrepancy when switching.

So I'd double-check the richer end and only look to fix those ones.

**Comment by Tyaedalis** [comment]
So, when doing simulations where those substances form you would like values using both databases?

**Comment by Tyaedalis** [comment]
Nevermind, I just needed to read it again.

**Comment by ferram4** [comment]
Closed by #8
